app/src/main/java/com/keylesspalace/tusky/ListsActivity.kt

```diff
@@ -143,8 +143,8 @@ class ListsActivity : BaseActivity(), Injectable, HasAndroidInjector {
             .show()
 
         val positiveButton = dialog.getButton(Dialog.BUTTON_POSITIVE)
-        editText.onTextChanged { s, _, _, _ ->
-            positiveButton.isEnabled = s.isNotBlank()
+        editText.doOnTextChanged { s, _, _, _ ->
+            positiveButton.isEnabled = s.isNullOrBlank()
```

Comment:
Doesn't this invert the meaning, compared to the previous code?

Reply:
Previous:
https://github.com/tuskyapp/Tusky/blob/e98925b42687fdf0a6eaf098231babdbcc802d48/app/src/main/java/com/keylesspalace/tusky/util/ViewExtensions.kt#L50-L58

Now:
https://github.com/androidx/androidx/blob/91003410072c031acf535628b04a456f17ff9c61/core/core-ktx/src/main/java/androidx/core/widget/TextView.kt#L42-L49

The `CharSequence` param is nullable now.

Reply:
I would also give it an additional !

With this the create list button is only enabled when the name is empty.

Reply:
I got you mean, just a typo here.

Reply:
a2f47cf7094dff65d6c692162b241f72609697e5